src/Microsoft.AspNetCore.SignalR.Core/Hub`T.cs

```diff
@@ -5,10 +5,17 @@
 
 namespace Microsoft.AspNetCore.SignalR
 {
+    /// <summary>
+    /// A base class for a strongly typed SignalR hub.
```

Comment:
A base class for a SignalR hub that uses an interface to represent client invocations.